spec/controllers/issues/send_issue_spec.rb

```diff
@@ -0,0 +1,49 @@
+describe IssuesController, type: :controller do
+  describe 'POST :send_issue' do
+    before { sign_in(manager) }
+
+    let(:manager) { create :manager }
+    let(:issue) { create :issue }
+    let!(:link) { create :link }
+    let!(:subscriber) { create :subscriber }
+
+    context 'when issue is not sent' do
+      it 'sends issue' do
+        post :send_issue, id: issue.id
+
+        issue.reload
+        link.reload
+
+        expect(assigns(:issue)).to eq issue
+        expect(issue.sent_at).not_to be_nil
+        expect(link.issue_id).to eq issue.id
+        expect(ActionMailer::Base.deliveries.count).to eq 1
+
+        expect(subject).to redirect_to issues_path
+        expect(response.status).to eq 302
+      end
+    end
+
+    context 'when issue is already sent' do
+      it 'redirects to issues' do
+        issue.update(sent_at: Time.zone.now)
```

Comment:
as you used `let()` to create an issue, it's better just to redefine `issue` here before `it` like
```ruby
  let(:issue) { create :issue, sent_at: Time.zone.now }
```